File: crates/tldr-cli/tests/fixtures/vuln_migration_v1/c/path_traversal_string_literal_fp.c
/* vt=PathTraversal lang=c — names below are inside strings/comments only
 *
 * getenv, mysql_query, system, fopen — referenced in comments below.
 */
#include <stdio.h>

const char *docs(void) {
    const char *s = "getenv -> mysql_query, system, fopen — none invoked here";
    return s;
}

File: crates/tldr-cli/tests/fixtures/vuln_migration_v1/rust/path_traversal_string_literal_fp.rs
//! vt=PathTraversal lang=rust — names below are inside strings/comments only
//!
//! Documentation strings only. None of `std::env::var`,
//! `std::process::Command`, `std::fs::File::open`, `reqwest::blocking::get`,
//! or `serde_json::from_str` are invoked below.

pub fn docs_only() -> String {
    let doc = "std::env::var(\"X\") flows into std::process::Command and std::fs::File::open";
    let more = "reqwest::blocking::get and serde_json::from_str — referenced in strings only";
    format!("{} {}", doc, more)
}
